apps/src/StudioApp.js

```diff
@@ -34,7 +34,11 @@ var annotationList = require('./acemode/annotationList');
 var shareWarnings = require('./shareWarnings');
 import { setPageConstants } from './redux/pageConstants';
 import { lockContainedLevelAnswers } from './code-studio/levels/codeStudioLevels';
-import SmallFooter from '@cdo/apps/code-studio/components/SmallFooter';
+import SmallFooter from './code-studio/components/SmallFooter';
+import project from './code-studio/initApp/project';
+import assets from './code-studio/assets';
+import i18n from './code-studio/i18n';
```

Comment:
yep. But no reason to use them outside of tests where you don't have to add a bunch of `../../../` all over the place.